CourseManager/Views/ExampleViews/MatchedGeometryEffect/Views/SofaListView.swift

```diff
@@ -0,0 +1,83 @@
+//
+//  SofaListView.swift
+//  CourseManager
+//
+//  Created by Summer Crow on 2022-12-08.
+//
+
+import SwiftUI
+
+struct SofaListView: View {
+    @Namespace var namespace
+    @State var selectedID = UUID()
```

Comment:
It is better to store here optional instead of default. The problem is that by construction of the UUID(), you will create on every view construction a new object. However, the way, in which the view will react on this is quite questionable
In general, it seems, that the SwiftUI will handle such thing, because of view identification and state storage, but it is not easy to see

Reply:
Makes sense, updated